libs/fluid-elements/checkbox/src/lib/checkbox.scss

```diff
@@ -0,0 +1,102 @@
+@import '~@dynatrace/fluid-design-tokens/global/spacing';
+@import '~@dynatrace/fluid-design-tokens/global/typography';
+
+:host {
```

Comment:
I think we should do something similar to the button here and declare intermediate css custom properties. 
https://github.com/dynatrace-oss/barista/blob/master/libs/fluid-elements/button/src/lib/button.scss

Reply:
But i am fine with doing that in a followup issue if you like. Please create an issue in that case :)

Reply:
Issue created at #1191